### DISCUSSION TITLE
general help , unable to install exemple.yaml   -> indentation error ?

### DISCUSSION THREAD

**Discussion by tfoutfou** [omerfaruk-aran/esphome_samsung_hvac_bus#170] [General]
Hi ,
quick presentation ,
long time Arduino DIYer and home automation DYIer , i have managed to do pretty much whatever i want and more.
but have never used ESPHome or Home Assistant

have been using a device to read SAMSUNG AC , made by "INTESIS" , convert NON-NASA to RS485 MODBUS
i can read the MODBUS register and get every interresting value and control the HVAC , no problem , expect the box just died and cost too much , 250+$

i am looking for free DYI way , so this repository seem to be OK

so far i have :
-> installed ESPHOME , both on linux or windows , both runing OK
-> flashed , connected and adopted my first ESP32
-> create new device and copy exemple.yaml , replaced wifi credentials , OTA

tried to install this config and get error about "indentation" , tried to fix for HOURS and HOURS , dont know what i am doing wrong

```
[non_nasa_keepalive] is an invalid option for [samsung_ac]. Please check the indentation.
[capabilities] is an invalid option for [samsung_ac]. Please check the indentation.
```

(cant post file it dont show up nicely)
but i downloaded the exemple and just modified wifi credentials

Please help , it must be a stupid error

**Comment by lanwin** [comment]
Hey, you need to change the soruce from github://lanwin/esphome_samsung_ac@stable to github://lanwin/esphome_samsung_ac@main. 

The example yaml mostly reflects the main branch.

**Comment by tfoutfou** [comment]
i dont understand why but it compile now !

thanks

will probably come back for further question when i get it wired and runing
